presto-main/src/main/java/io/prestosql/sql/analyzer/QueryExplainer.java

```diff
@@ -178,7 +179,7 @@ public Plan getLogicalPlan(Session session, Statement statement, List<Expression
 
         // plan statement
         LogicalPlanner logicalPlanner = new LogicalPlanner(session, planOptimizers, idAllocator, metadata, new TypeAnalyzer(sqlParser, metadata), statsCalculator, costCalculator, warningCollector);
-        return logicalPlanner.plan(analysis);
+        return logicalPlanner.plan(analysis, OPTIMIZED_AND_VALIDATED, true);
```

Comment:
can we remove `io.prestosql.sql.planner.LogicalPlanner#plan(io.prestosql.sql.analyzer.Analysis)`?

Reply:
right